torch/jit/frontend.py

```diff
@@ -285,6 +285,8 @@ def build_AnnAssign(ctx, stmt):
 
     @staticmethod
     def build_Delete(ctx, stmt):
+        if len(stmt.targets) > 1:
+            raise UnsupportedNodeError(ctx, stmt, reason='with more than 1 operands')
```

Comment:
Since `del` is partially supported, I think it would be more appropriate to use `NotSupportedError` here instead of `UnsupportedNodeError`.

Reply:
Done.